lib/overcommit/git_config.rb

```diff
@@ -0,0 +1,12 @@
+module Overcommit
+  # Get configuration options from git
+  module GitConfig
+    module_function
+
+    def comment_character(options = '')
```

Comment:
Typically `options` is the name given to an options hash argument.

Since this option is used only within the context of tests, I don't think it makes sense to have here. Instead, we should write the tests to set the option in `before` blocks (you can create a one-off repo using the `repo` helper so you can modify the repo's `commentchar` option instead of your global one).